string cube_colour
float64 normalisedCoordinateX
float64 normalisedCoordinateY
float64 area
